### PR TITLE
Implement Credential hashes API

### PR DESCRIPTION
**Pull request by ashkarpetin**
## Summary
[Preview](https://preview.redoc.ly/rebilly-users/add-credential-hash-api/tag/Credential-hashes)

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/openapi.yaml [hunk 1]
```diff
@@ -352,6 +352,8 @@ tags:
       Rebilly hosted checkout forms provide secure and compliant checkouts.
       Checkout forms are customizable, and use fully responsive design,
       built-in error messaging, validation, and expedited checkout for returning customers.
+  - name: Credential hashes
+    description: Use these operations to get, create, and manage credential hashes.
```
Comment (adamaltman) [inline openapi/openapi.yaml:356]:
```suggestion
    description: Use these operations to get, create, and manage credentials for third-party services.
```

**Comment by adamaltman** [comment]
This is good. How about also marking the older paths/operations as deprecated?

**Comment by ashkarpetin** [comment]
Need to implement GetCredentialHashItemCollection:

https://github.com/Rebilly/api-definitions/blob/ac98d71ae8c7a653d92202107b2cdf59b5a8a1ef/openapi/paths/credential-hashes%40oauth2%40%7Bhash%7D%40items.yaml#L10

**Comment by ashkarpetin** [comment]
[Avalara](https://github.com/Rebilly/api-definitions/pull/1102) would go there to

**Pull request closed without merging**